cloud-sql/mysql/sqlalchemy/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 # Use the official Python image.
 # https://hub.docker.com/_/python
-FROM python:3
+FROM python:3.11
```

Comment:
Can you verify if 3.11 is available in the image you're installing?

Reply:
Yes, it is available. I've checked by ensuring the `docker build .` works. It is currently broken with `python:3`.

Another sample which uses this image - https://github.com/GoogleCloudPlatform/python-docs-samples/blob/68fa1ecd8027a216ffc2e5cd0fa3956b6c852b9f/run/idp-sql/Dockerfile#L7